firstvoices/backend/views/site_feature_views.py

```diff
@@ -37,7 +40,8 @@
         ],
     ),
     create=extend_schema(
-        description="Create a new feature flag for the site. Site feature keys cannot be changed after creation.",
+        description="Create a new feature flag for the site. Site feature keys cannot be changed after creation. "
+        "Triggers a reindex of all media content in the site.",
```

Comment:
All of our APIs keep the content up to date in the search indices, so I don't think we need to mention this specifically in the docs.